### DISCUSSION TITLE
✅ Posts Widget - Text Hover

### DISCUSSION THREAD

**Discussion by crispee2000** [elementor/elementor#21922] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I'm not seeing any way to add a Title link hover effect in the Style tab like it is in other widgets.  Need to be able to set this without CSS.

### Describe the solution you'd like

Add Hover tab to Title section in Style tab and then Content.

### Describe alternatives you've considered

You can do by CSS but...

**Comment by nicholaszein** [comment]
Hi @crispee2000!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor  Pro v3.26.0! 🥳 

You can now define hover colors for the **Heading** widget, and use it inside a **Loop Item** template to define the hover color for your post titles in the **Loop Grid** widget! 💪 

If you haven't already switched to Containers and Loop Grids, now it's the time. Unleash the power of these new features and build your own Posts layout with all the control you need over your designs!

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
